src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../../api/people';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+interface Error {
+  status: boolean,
+  message: string,
+}
+
+const defaultError = {
+  status: false,
+  message: '',
+};
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState<Error>(defaultError);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const getPeopleFromServer = async () => {
+    try {
+      const peopleFromServer = await getPeople();
+
+      setPeople(peopleFromServer);
+      setError(defaultError);
+    } catch (e: any) {
+      setError({
+        status: true,
+        message: e.message,
+      });
```

Comment:
Don't use error message here, better provide your custom message. Then you can skip `e` argument.